site/content/docs/user/known-issues.md

```diff
@@ -298,6 +298,10 @@ systemctl restart firewalld
 
 See [#1547 (comment)](https://github.com/kubernetes-sigs/kind/issues/1547#issuecomment-623756313)
 
+## Fedora32 and Docker
+
+The situation with Docker on Fedora is not ideal. See Fedora Magazine article ["Fedora and Docker"](https://fedoramagazine.org/docker-and-fedora-32/) for information regarding workarounds for cgroups, firewalld, and installing docker.
```

Comment:
thanks for the PR, requesting a small rewording:

```suggestion
Docker is currently not supported on Fedora 32 and requires additional setup. See Fedora Magazine article ["Fedora and Docker"](https://fedoramagazine.org/docker-and-fedora-32/) for information regarding workarounds for cgroups, firewalld, and installing Docker.
```

p.s. please keep the commits squashed to 1.

Reply:
- cgroupsv2 broke kubernetes not just docker / kind
- firewalld & fedora 32 already has an entry
- docker being broken on fedora is a fedora issue, fedora broke a lot of things with these changes